### PR TITLE
Play the gif only on hover and show static image when non hovered

### LINKED ISSUE #54: Syncing screen change in Teal Demo App page
### Feature description

On https://insightsengineering.github.io/teal.gallery/demo.html webpage, demo apps appears in a grid with a clickable gif image. However, the speed at which the gif changes or move between tabs are different between panels, so for webpage viewers, it's very messy with various parts of the webpage seems to be flashing at varying speed.

Suggest to improve the coordination and sync the "screen movement" to some consistent interval across all apps. For example, switching tabs every 2 sec for all apps, and maybe avoid scrolling actions all together.

As we have quite a few presentations/workshops upcoming, please consider prioritizing this issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by vedhav**
Closes #54

### CODE REVIEW

**Comment by vedhav** [comment]
Note: Will have to update all the GIF recordings before merging.

@lcd2yyz We are not limited to showing just the first tab for the static image, We can show the most attractive tab for each teal app when the card is not hovered. Do you have a recommendation on which tab we should show for each of the nine apps?

**Comment by lcd2yyz** [comment]
> We can show the most attractive tab for each teal app when the card is not hovered. Do you have a recommendation on which tab we should show for each of the nine apps?

@vedhav thanks for the great suggestion! My recommendation as below

`basic-teal` - NA
`exploratory` - Association
`safety` - Demographic table
`efficacy` - KM plot (but adjust to show one plot per page (instead of the default with two facet)
`patient-profile` - Adverse events
`early-dev` - Waterfall
`longitudinal` - Visualization - Correlation plot
`RNA-seq` - PCA
`python` - scatterplot

**Comment by lcd2yyz** [comment]
Thanks @vedhav for the quick work, looks really awesome now!
One minor change, I wasn't being very specific with the RNA-seq app (sorry!) I actually meant the "PCA -> PC & Sample Collection" with the heatmap plot. That definitely looks a lot more interesting than the PCA with 3 dots 🤣 

Please proceed with the new app recording to make it all nice and pretty 😃

File: _internal/quarto/assets/img/RNA-seq.png
Comment (lcd2yyz) [inline _internal/quarto/assets/img/RNA-seq.png:1]: Please change this to the PC and Sample Collection subtab! Sorry for not specifying it clearly

**Review by lcd2yyz — APPROVED** [review]
One minor change request

**Pull request merged**